src/views/DocumentationTopic.vue

```diff
@@ -121,7 +122,7 @@ export default {
     Nav: DocumentationNav,
     QuickNavigationModal,
   },
-  mixins: [performanceMetrics, onPageLoadScrollToFragment],
+  mixins: [performanceMetrics, onPageLoadScrollToFragment, OnThisPageRegistrator],
```

Comment:
My only concern with making this a mixin is that it is currently only used in a single component (I think). Was the reason for extracting it to avoid making this component really big with all of the extracted logic?

Reply:
Yeah, I thought about keeping things separate. Plus it should help third party clients integrate the mixin into their own View pages.